irods_audit_elk_stack/kibana/irods_dashboard.ndjson

```diff
@@ -1,9 +1,9 @@
_name\":[\"pep_auth_agent_auth_request_pre\",\"pep_api_authenticate_pre\"]}},\"language\":\"lucene\"},\"filter\":[],\"indexRefName\":\"kibanaSavedObjectMeta.searchSourceJSON.index\"}"},"title":"Connections Per Minute","uiStateJSON":"{}","version":1,"visState":"{\"title\":\"Connections Per Minute\",\"type\":\"line\",\"aggs\":[{\"id\":\"1\",\"enabled\":true,\"type\":\"count\",\"params\":{\"emptyAsNull\":false},\"schema\":\"metric\"},{\"id\":\"2\",\"enabled\":true,\"type\":\"date_histogram\",\"params\":{\"field\":\"@timestamp\",\"timeRange\":{\"from\":\"now-15m\",\"to\":\"now\"},\"useNormalizedEsInterval\":true,\"extendToTimeRange\":false,\"scaleMetricValues\":true,\"interval\":\"m\",\"used_interval\":\"1m\",\"drop_partials\":false,\"min_doc_count\":1,\"extended_bounds\":{},\"json\":\"\"},\"schema\":\"segment\"},{\"id\":\"3\",\"enabled\":true,\"type\":\"terms\",\"params\":{\"field\":\"hostname.keyword\",\"orderBy\":\"_key\",\"order\":\"desc\",\"size\":10,\"otherBucket\":false,\"otherBucketLabel\":\"Other\",\"missingBucket\":false,\"missingBucketLabel\":\"Missing\"},\"schema\":\"group\"}],\"params\":{\"addLegend\":true,\"addTimeMarker\":false,\"addTooltip\":true,\"defaultYExtents\":false,\"drawLinesBetweenPoints\":true,\"interpolate\":\"linear\",\"radiusRatio\":9,\"scale\":\"linear\",\"setYExtents\":false,\"shareYAxis\":true,\"showCircles\":true,\"smoothLines\":false,\"times\":[],\"yAxis\":{},\"type\":\"line\",\"grid\":{\"categoryLines\":false,\"style\":{\"color\":\"#eee\"}},\"categoryAxes\":[{\"id\":\"CategoryAxis-1\",\"type\":\"category\",\"position\":\"bottom\",\"show\":true,\"style\":{},\"scale\":{\"type\":\"linear\"},\"labels\":{\"show\":true,\"truncate\":100,\"filter\":true},\"title\":{}}],\"valueAxes\":[{\"id\":\"ValueAxis-1\",\"name\":\"LeftAxis-1\",\"type\":\"value\",\"position\":\"left\",\"show\":true,\"style\":{},\"scale\":{\"type\":\"linear\",\"mode\":\"normal\",\"setYExtents\":false,\"defaultYExtents\":false},\"labels\":{\"show\":true,\"rotate\":0,\"filter\":false,\"truncate\":100},\"title\":{\"text\":\"Count\"}}],\"seriesParams\":[{\"show\":\"true\",\"type\":\"line\",\"mode\":\"normal\",\"data\":{\"label\":\"Count\",\"id\":\"1\"},\"valueAxis\":\"ValueAxis-1\",\"drawLinesBetweenPoints\":true,\"showCircles\":true,\"interpolate\":\"linear\",\"radiusRatio\":9,\"circlesRadius\":1}],\"legendPosition\":\"right\",\"palette\":{\"type\":\"palette\",\"name\":\"kibana_palette\"},\"isVislibVis\":true,\"detailedTooltip\":true,\"fittingFunction\":\"linear\",\"legendSize\":\"auto\",\"truncateLegend\":true,\"maxLegendLines\":1,\"labels\":{},\"thresholdLine\":{\"show\":false,\"value\":10,\"width\":1,\"style\":\"full\",\"color\":\"#E7664C\"}}}"},"coreMigrationVersion":"8.3.1","id":"Connections-Per-Minute","migrationVersion":{"visualization":"8.3.0"},"references":[{"id":"irods-audit-pattern","name":"kibanaSavedObjectMeta.searchSourceJSON.index","type":"index-pattern"}],"type":"visualization","updated_at":"2022-07-12T19:15:22.518Z","version":"WzEwNCwxXQ=="}
```

Comment:
Point of pedantry: Should we change the title to "Authentication Requests Per Minute" rather than "Connections Per Minute"?

Reply:
mmm, discush.

we're counting these... `[\"pep_auth_agent_auth_request_pre\",\"pep_api_authenticate_pre\"]`

and... each is an incoming API hit... and I see... you're saying a long-running client could have multiple of these on a single connection.

yes, how does HTTP API appear here?

Reply:
> yes, how does HTTP API appear here?

Depends on the config of the HTTP API. If connection pooling is used, the PEPs for check_auth_creds and switch_user will be visible. If pooling is not used, you'll see the same PEPs produced by icommands.

Reply:
right, and does that vote for either the naming as is - or what alan proposed as a change?   or a third better naming option?

Reply:
Because iRODS separates establishment of connections (sockets) and auth, changing the title to include auth seems more correct.

Reply:
changed - resolving.